src/main/java/uk/gov/justice/probation/courtcaseservice/jpa/entity/CaseCommentEntity.java

```diff
@@ -0,0 +1,43 @@
+package uk.gov.justice.probation.courtcaseservice.jpa.entity;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.With;
+import lombok.experimental.SuperBuilder;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "CASE_COMMENTS")
+@AllArgsConstructor
+@NoArgsConstructor(access = AccessLevel.PROTECTED, force = true)
+@SuperBuilder
+@Getter
+@With
+@EqualsAndHashCode(callSuper = true)
+public class CaseCommentEntity extends BaseEntity {
+
+    @Id
+    @Column(name = "ID", updatable = false, nullable = false)
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @JsonIgnore
+    private final Long id;
+
+    @Column(name = "CASE_ID", updatable = false, nullable = false)
+    private final String caseId;
+
+    @Column(name = "COMMENT_ID", updatable = false, nullable = false)
+    private final String commentId;
```

Comment:
Why is there a separate commentId? Is this just so we don't expose our internal database keys?

Reply:
That one too and it is not a good practice to use database generated PKs to use as resource ids. Though it should not matter in this case as we use case id access these always. Will update to use the id colum

Reply:
I was just curious here, I wasn't suggesting you change it. You have a point about not exposing the database keys but wasn't sure what this was being used for. I suppose this getting a step ahead for when we need to start deleting them which is a good call.